security/client/ipsec.go

```diff
@@ -65,6 +65,14 @@ func do_ipsec(conn grpc.ClientConnInterface, ctx context.Context) {
 	log.Printf("Release [%v]", vresp.GetRelease())
 	log.Printf("Machine [%v]", vresp.GetMachine())
 
+	stats_req := pb.IPsecStatsReq {}
```

Comment:
should `IPsecStatsReq` receive any argument ? if you want stats only for specific UUID, for example ?

Reply:
I think we will evolve it that way for sure, but for now, the API requires no parameter on the way in.